styles.css

```diff
@@ -0,0 +1,93 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.main {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  background-image: url("background.png");
+  background-repeat: no-repeat;
+}
+
+.logo {
+  position: absolute;
+  top: 40px;
+  font-size: 18px;
+  font-weight: 700;
+  padding: 24px;
+  color: #6070ff;
+  text-decoration: none;
+}
+
+.navbar {
+  position: absolute;
+  top: 30px;
+  right: 30px;
+  padding: 36px;
+  color: #acb7c3;
+}
+
+.primarytext {
+  width: 327px;
+  height: 150px;
+  margin-top: 206px;
+  margin-left: 24px;
+  margin-right: 24px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.text2 {
+  width: 327px;
+  height: 146px;
+  margin-left: 24px;
+  margin-right: 24px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.connect {
+  width: 200px;
+  height: 24px;
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 24px;
+  color: #7f8cff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.socialm {
+  width: 200.17px;
+  height: 20px;
+}
+
+ul li {
+  display: inline;
+  list-style-type: none;
+  padding: 5px;
+  color: #acb7c3;
```

Comment:
- [ ] For `lets connect` please add the correct margin-left value to match the given Figma design.
- [ ] For the social icons Kindly make them smaller a little bit and add the correct color to them and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170528083-77669655-77d9-487d-87b7-d1a28030c88c.png)  | ![image](https://user-images.githubusercontent.com/68796232/170528112-967aaece-5581-43c9-913a-131bcaf9a5fa.png)  |